From 24bb285a2a07ea15a98e8002810304f8a0e4d8a2 Mon Sep 17 00:00:00 2001
From: Mikhail Zolotukhin <mail@gikari.com>
Date: Thu, 17 Nov 2022 21:09:18 +0100
Subject: [PATCH] fix: use current project directory for icons install

---
 src/gui/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gui/CMakeLists.txt b/src/gui/CMakeLists.txt
index a48cf47..4063ce3 100644
--- a/src/gui/CMakeLists.txt
+++ b/src/gui/CMakeLists.txt
@@ -101,10 +101,10 @@ if(NOT APPLE)
 	install(FILES ${PROJECT_SOURCE_DIR}/LICENSE
 		DESTINATION ${CMAKE_INSTALL_DATADIR}/nvim-qt/)
 	install(FILES nvim-qt.desktop DESTINATION ${CMAKE_INSTALL_DATADIR}/applications)
-	install(FILES ${CMAKE_SOURCE_DIR}/third-party/neovim.png
+    install(FILES ${PROJECT_SOURCE_DIR}/third-party/neovim.png
 			RENAME nvim-qt.png
 			DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/192x192/apps/)
-	install(FILES ${CMAKE_SOURCE_DIR}/third-party/neovim.svg
+    install(FILES ${PROJECT_SOURCE_DIR}/third-party/neovim.svg
 			RENAME nvim-qt.svg
 			DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/scalable/apps/)
 endif()
-- 
2.38.1.windows.1

